x-pack/plugins/telemetry/server/plugin.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { CoreSetup } from 'src/core/server';
+import { registerRoutes } from './routes';
+
+export class Plugin {
```

Comment:
Unless there's some reason it **HAS** to be named `Plugin`, it would be useful to have a descriptive name, for debugging purposes (debugging, profiling, heap snapshots, etc).